src/translations/screens/TicketAssistant.ts

```diff
@@ -10,22 +10,206 @@ const TicketAssistantTexts = {
+      'You will travel for more than half a year.',
+    ),
+    mainButton: _('Neste', 'Next'),
+    a11yNextPageHint: _(
+      'Aktiver for å gå til neste side',
+      'Activate to go to next page',
+    ),
+    a11ySliderHint(amount: {value: number}) {
+      return _(
+        `Du skal reise i ${amount.value} dager`,
+        `You will travel for ${amount.value} days`,
+      );
+    },
+  },
+  summary: {
+    title: _('Anbefalt for', 'Recommended for'),
+    titleA11yLabel: _('Anbefalt for', 'Recommended for'),
+    description: (data: {frequency: number; date: string}) =>
+      _(
+        `${data.frequency} turer i uka fram til ${data.date}`,
+        `${data.frequency} trips per week until ${data.date}`,
+      ),
+    savings: (data: {
+      totalSavings: number;
+      perTripSavings: string;
+      alternative: string;
+    }) =>
+      _(
+        `Da sparer du ${data.totalSavings} kr totalt (${data.perTripSavings} kr per tur) sammenlignet med ${data.alternative} enkeltbilletter`,
+        `You save ${data.totalSavings} kr in total (${data.perTripSavings} kr per trip) \n compared to ${data.alternative} single tickets`,
+      ),
+    savingsA11yLabel: (data: {
+      totalSavings: number;
+      perTripSavings: string;
+      alternative: string;
+    }) =>
+      _(
+        `Da sparer du ${data.totalSavings} kr totalt (${data.perTripSavings} kr per tur) sammenlignet med ${data.alternative} enkeltbilletter`,
+        `You save ${data.totalSavings} kr in total (${data.perTripSavings} kr per trip) compared to ${data.alternative} single tickets`,
+      ),
+    traveller: _('Reisende', 'Traveller'),
+    zones: _('Soner', 'Zones'),
+    price: _('Pris', 'Price'),
+    pricePerTrip: _('Pris per tur:', 'Price per trip:'),
+    resultMoreThan180Days: _(
+      'Du skal reise i mer enn et halvt år.',
+      'You will travel for more than half a year.',
+    ),
+    mainButton: _('Lukk', 'Close'),
+    buyButton: _('Kjøp denne billetten', 'Buy this ticket'),
+    a11yBuyButtonHint: _(
+      'Aktiver for å gå til siden hvor du kan kjøpe denne billetten',
+      'Activate to go to the page where you can buy this ticket',
+    ),
+    ticketSummaryA11yLabel: (data: {
+      ticket: string;
+      traveller: string;
+      tariffZones: string;
+      price: string;
+      pricePerTrip: string;
+    }) =>
+      _(
+        `Du har valgt ${data.ticket}. \n Reisende: ${data.traveller}. \n Soner: ${data.tariffZones}. \n Pris: ${data.price}kr. \n Pris per tur: ${data.pricePerTrip}kr.`,
+        `You have selected ${data.ticket}. \n Traveller: ${data.traveller}. \n Zones: ${data.tariffZones}. \n Price: ${data.price}. \n Price per trip: ${data.pricePerTrip}.`,
+      ),
+    durationNotice: _(
+      'NB: denne billetten dekker ikke hele perioden du har satt. Ta veilederen på nytt når denne billetten utløper.',
+      'NB: this ticket does not cover the entire period you have set. Take the guide again when this ticket expires.',
```

Comment:
```suggestion
      'NB: this ticket does not cover the entire period you have set. Take the assistant again when this ticket expires.',
```
You have used the wording assistant everywhere but here and line 193, maybe change to be consistent?